src/eredis_sync.erl

```diff
@@ -15,32 +15,35 @@
 -type response() :: {ok, iodata()} | {error, error()}.
 -type response_list() :: [response()].
 
--spec connect(inet:ip_address(), inet:port_number()) -> {ok , conn()} | {error, error()}.
+-spec connect(inet:ip_address() | string(), inet:port_number()) -> {ok , eredis_sync:conn()} | {error, error()}.
 
 connect(Host, Port) ->
   connect(Host, Port, ?DEFAULT_TIMEOUT).
 
--spec connect(inet:ip_address(), inet:port_number(), timeout()) -> {ok , conn()} | {error, error()}.
+-spec connect(inet:ip_address() | string(), inet:port_number(), timeout()) -> {ok , eredis_sync:conn()} | {error, error()}.
```

Comment:
`string()`  does not denote binary in Erlang

Reply:
I changed it to iodata